### PR TITLE
setup,doc: remove centos5

### PR DESCRIPTION
**Pull request by maclover7**
As discussed during the last WG meeting, these machines will be retired
shortly, so their corresponding playbook + docs can be removed as well.

### CODE REVIEW

File: doc/non-ansible-configuration-notes.md [hunk 1]
```diff
@@ -130,63 +130,6 @@ Then, manually:
 * Update <https://github.com/nodejs/build/blob/master/ansible/inventory.yml> to reflect any host additions or changes (this is primarily for automatic .ssh/config setup purposes currently)
 * Run Ansible from <https://github.com/nodejs/build/tree/master/setup/raspberry-pi> with `ansible-playbook ansible-playbook.yaml -i ../ansible-inventory --limit <hostname>`
 
-## git on CentOS 5
```
Comment (refack) [inline doc/non-ansible-configuration-notes.md]: Maybe keep this for the time we are maintaining the machine manually.
Or even leave a "scar" pointing to this changeset (https://github.com/nodejs/build/pull/1454/files)

Comment (maclover7) [reply doc/non-ansible-configuration-notes.md]: @refack What's a "scar"?

Comment (refack) [reply doc/non-ansible-configuration-notes.md]: Metaphone for something that is left after a resection... Something like:
```
## CentOS 5
(removed in https://github.com/nodejs/build/pull/1454/files)
```

**Comment by sam-github** [comment]
@AshCripps deleting unused config is worthwhile, can you take a look at this and see if the changes suggested apply to master? If so, it should be updated and applied. If not, then it can be closed. Thanks.

**Comment by AshCripps** [comment]
@sam-github I don't think these files exist anymore anyway? at least searching for `centos5` bring up nothing

**Pull request closed without merging**